# mongodb settings
use_mongo_db: False
mongodb:
  db_name: walk_performance
  db_host: localhost
  db_port: 27018
  drop_old: False # WARNING: This will kill all data permamently!

# logger configs
logger:
  contact_force_z_thresh: 0.0

# monitor configs
monitor:
  max_step_difference: 0.2
  max_yaw_difference: 0.2

